assessments/projects/longhorn/self-assessment.md

```diff
@@ -0,0 +1,442 @@
+
+
+# Self-assessment
+This is the Longhorn's Self-assessment documentation that we wrote after we began thinking about the security of the project, determining gaps in its security, and preparing security documentation for Longhorn's users. This project is currently in the CNCF Incubation phase preparing for Graduation.
+
+# Self-assessment outline
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Threat Modelling](#Threat-Modelling)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+* [Action items](#Action-Items)
+
+## Metadata
+
+A table at the top for quick reference information, later used for indexing.
```

Comment:
```suggestion
```

Reply:
Please check to ensure that no text from the template is copied over into this document